lib/ohai/plugins/filesystem.rb

```diff
@@ -154,11 +154,21 @@ def run_with_check(bin, &block)
 
     # Grab filesystem data from df
     run_with_check("df") do
-      so = shell_out("df -P")
+      # df runs statfs across all filesystems including remote ones. When run
+      # at scale, Ohai adds extra load, simply to obtain block / inode usage
+      # free information. This is rarely useful. Turn on this option to
+      # disable remote filesystem enumeration in df. Note that device +
+      # mountpoint will still be obtained from mount and /proc/mounts.
+      params = if Ohai.config[:plugin][:filesystem][:no_df_remote]
+        ' -l'
```

Comment:
FYI this should be aligned like:

```
params = if Ohai.config[...
           ' -l'
         else
           ''
         end
```

Which is terrible, so instead use terniary:

```
params = Ohai.config[:plugin][:filesystem][:no_df_remote] ? ' -l' : ''
```

Reply:
My original code used the long option. I think it's been supported for years. I much prefer it, especially in code. However, this leads it to be inconsistent with the `-P` / `--portability` option and all things considered, I want to keep the style close to the original.

My original code exceeded 80 columns, so I split it across lines using the longer if. Now the option is short, it all fits.